cmd/process.go

```diff
@@ -235,6 +254,10 @@ func validateFlags() error {
 		return errRequiredTopics
 	}
 
+	if config.AppConfig.ReconcilerInterval <= 0 {
+		return errInvalidInterval
+	}
+
```

Comment:
you'll want to put this higher up in the function to bail earlier. The go routine running the reconciler is already started at this point